v2/oapi/oapi.gen.go

```diff
@@ -2968,7 +4257,12 @@ type TemplateVisibility string
 
 // Zone
 type Zone struct {
-	Name *ZoneName `json:"name,omitempty"`
+	// Zone API endpoint
+	ApiEndpoint *string   `json:"api-endpoint,omitempty"`
+	Name        *ZoneName `json:"name,omitempty"`
+
+	// Zone SOS endpoint
+	SosEndpoint *string `json:"sos-endpoint,omitempty"`
```

Comment:
For those, you can probably create an `Endpoint` type statically in the same package? To be compatible with the generator, in case we generate a last time again :)

Reply:
I can of course. It's a quick patch this time as I hope we will not need to sync v2 generator code again.